packages/palette/src/elements/Cards/LargeCard.ios.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react"
+import { Image, View } from "react-native"
+import LinearGradient from "react-native-linear-gradient"
+import { color, space } from "../../helpers"
+import { Flex } from "../Flex"
+import { Spacer } from "../Spacer"
+import { Sans } from "../Typography"
+import { Tag } from "./CardTag.ios"
+import { LargeCardProps } from "./LargeCard.shared"
+
+/**
+ * `Large` is a card with one image one tall image, and text for title and subtitle
+ * at the bottom.
+ */
+export const LargeCard: React.FC<LargeCardProps> = ({
+  image,
+  title,
+  subtitle,
+  tag,
+}) => {
+  return (
+    <View
+      style={{ width: "100%", aspectRatio: 1.0 / 1.33, flexDirection: "row" }}
+    >
+      <Flex flex={2} background={color("black10")}>
+        <Image source={{ uri: image }} style={{ flex: 1 }} />
+      </Flex>
+      <LinearGradient
+        colors={["rgba(0, 0, 0, 0)", "rgba(0, 0, 0, 1)"]}
+        style={{
+          position: "absolute",
+          width: "100%",
+          height: "100%",
+          opacity: 0.15,
+        }}
+      />
+      <Flex
+        style={{
+          position: "absolute",
+          bottom: 0,
+          left: space(1),
+          right: space(6),
```

Comment:
would it make sense to use `maxWidth` here instead? Since the card is responsive, I'd assume we want a fixed break point on iPad.

Reply:
Hm, let me try it there. My idea was that there will always be a grid or a list that will allocate some width to each item, and them the card will just occupy that space. 🤔